pkg/vsphere/vm/vm.go

```diff
@@ -803,9 +788,7 @@ func (vm *VirtualMachine) PowerOn(op trace.Operation) error {
 		return vm.powerOnDRS(op)
 	}
 
-	// TODO(jzt): uncomment this once we have CheckHost implemented.
-	// see https://github.com/vmware/vic/issues/7654
-	// op.Warnf("Host is not adequate for power-on, getting placement recommendation")
+	op.Debugf("Host is not adequate for power-on, getting placement recommendation")
```

Comment:
I know this is likely a slippery slope. but since we are in debug should we log any additional information surrounding the host in this message?

Reply:
Yeah I could do that.. the host gets logged when attempting to relocate but it wouldn't hurt to have it here.